ortools/flatzinc/ortools.msc.in

```diff
@@ -0,0 +1,17 @@
+{
+  "id": "com.google.or-tools",
+  "name": "OR Tools",
+  "description": "Google's Operations Research tools FlatZinc interface",
+  "version": "${ortools_VERSION_MAJOR}.${ortools_VERSION_MINOR}.${ortools_VERSION_PATCH}",
+  "mznlib": "../ortools",
+  "executable": "${FZ_REL_INSTALL_BINARY}",
+  "tags": ["ortools", "cp","lcg","int"],
+  "stdFlags": ["-a","-f","-n","-p","-r","-s","-v"],
```

Comment:
supports -l too

Reply:
`-l` is not one of the standard flags described by MiniZinc (https://www.minizinc.org/doc-2.5.3/en/fzn-spec.html#solver-configuration-files), but we could add it under `extraFlags` in the configuration.

I had only looked at `--help` before, but I now see that `--helpfull` reveals many more flags that could potentially be added as extra flags. I'll have a look at adding them tomorrow.